### PR TITLE
Create FUNDING.yml

### PR DESCRIPTION
**Pull request by ja1m0n82**

### CODE REVIEW

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# These are supported funding ⁹model platforms
+
+github: # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
+patreon: # Replace with a single Patreon username
+open_collective: # Replace with a single Open Collective username
+ko_fi: # Replace with a single Ko-fi username
+tidelift: # Replace with a single Tidelift platform-name/package-name e.g., npm/babel
+community_bridge: # Replace with a single Community Bridge project-name e.g., cloud-foundry
+liberapay: # Replace with a single Liberapay username
```
Comment (deaths888kiss) [inline .github/FUNDING.yml:9]: https://api.paypal.com/v3/vault/payment-tokens

**Review by deaths888kiss — COMMENTED** [review]
https://api.paypal.com/v3/vault/payment-tokens

**Comment by devchristina** [comment]
Spam

**Comment by deaths888kiss** [comment]
??

**Comment by devchristina** [comment]
Unfortunately, @deaths888kiss. This repo was never intended to take on open source contributions from the community, therefore it was marked as Spam. Apologies for the impression it gave when we didn't have that feature turned off accordingly.

**Pull request closed without merging**